content/influxdb/cloud-serverless/admin/billing/limits.md

```diff
@@ -83,8 +76,10 @@ If you need higher storage-level limits, [contact InfluxData Sales](https://www.
 
 - **Data-in**: Rate of 5 MB per 5 minutes (average of 17 kb/s)
   - Uncompressed bytes of normalized [line protocol](/influxdb/cloud-serverless/reference/syntax/line-protocol/)
-- **Read**: Rate of 300 MB per 5 minutes (average of 1000 kb/s)
-  - Bytes in HTTP in response payload
+- **Read**:
+  - **HTTP response payload rate**: 300 MB data per 5 minutes (average of 1000 kb/s)
+  - **Partitions per query**: 500
```

Comment:
I don't know if you want to indicate that these are v3 only? It probably doesn't matter as users will only care when they hit the limits.

Reply:
@mhilton Are the limits in effect now? Can you link your PR here? Thanks!

Reply:
> @mhilton Are the limits in effect now? Can you link your PR here? Thanks!

They are in effect for new users. They haven't been enabled for existing users yet (waiting to get the go-ahead). The relevant PRs are:

 - https://github.com/influxdata/idpe/pull/18527
 - https://github.com/influxdata/quartz/pull/7535
 - https://github.com/influxdata/ui/pull/6875